docs/4.3/cli-docs.md

```diff
@@ -127,10 +127,13 @@ Run shell or execute a command on a remote SSH node
 | `-A, --forward-agent` | none | none | Forward agent to target node like `ssh -A`
 | `-L, --forward` | none | none | Forward localhost connections to remote server
 | `-D, --dynamic-forward ` | none | none | Forward localhost connections to remote server using SOCKS5
+| `-N, -no-remote-exec` | none | none | Don't execute remote command, useful for port forwarding
 | `--local` | none |  | Execute command on localhost after connecting to SSH node
 | `-t, --tty` | `file` |  | Allocate TTY
 | `--cluster` | none |  | Specify the cluster to connect
 | `-o, --option` | `local` |  |  OpenSSH options in the format used in the configuration file
+| `--enable-escape-sequences` | false | true/false | Enable support for SSH escape sequences. Type '~?' during an SSH session to list supported sequences.
+| `--use-local-ssh-agent` | none | true/false | Load generated SSH certificates into the local ssh-agent (specified via `$SSH_AUTH_SOCK`). You can also set `TELEPORT_USE_LOCAL_SSH_AGENT` environment variable. Default is true.
```

Comment:
This isn't actually how this flag works and is a bit confusing. The default is `--use-local-ssh-agent` - to disable the functionality, you actually have to pass `--no-use-local-ssh-agent`.

I _believe_ that the environment variable works the way you've written.

@awly Please confirm?

Reply:
Yeah, our flag parsing library doesn't handle `true/false` values for boolean flags, only `--flag/--no-flag`.
And yes, env vars need `true/false` values, opposite to flags.

Reply:
Thanks for the clarification, I simply removed the default / allowed values.